src/System.Transactions.Local/src/System/Transactions/DurableEnlistmentState.cs

```diff
@@ -11,128 +11,29 @@ namespace System.Transactions
     internal abstract class DurableEnlistmentState : EnlistmentState
     {
         // Double-checked locking pattern requires volatile for read/write synchronization
```

Comment:
The comment is in this path corefx/src/System.Transactions.Local/src/System/Transactions/VolatileEnlistmentState.cs which a recently added file that came in after I did merge base... hmmm some more LazyInitializer.EnsureInitialized to use...